docs/user/apirule-migration/01-84-migrate-oauth2-v1beta1-to-v2.md

```diff
@@ -18,6 +18,11 @@ APIRule in version `v1beta1` is deprecated and scheduled for removal. Once the A
 ## Steps
 
 In this example, the APIRule `v1beta1` was created with the **oauth2_introspection** handler, so the migration targets an APIRule `v2` using the **extAuth** handler. To illustrate the migration, the HTTPBin service is used, exposing the `/anything` and `/.*` endpoints. The HTTPBin service is deployed in its own namespace, with Istio enabled, ensuring the workload is part of the Istio service mesh.
+This example assumes that the targeted workload is only exposed by a single APIRule in version `v1beta1`.
+
+> [!WARNING]
+> If multiple APIRules target the same workload, you must perform an additional migration step to avoid traffic disruption. This step involves creating an additional, temporary AuthorizationPolicy before applying first migrated APIRule `v2` to allow traffic for workloads exposed by APIRules `v1beta1` not yet migrated to version `v2`. For detailed instructions, refer to the [Migrate multiple APIRules `v1beta1` targeting same workload](./01-90-migrate-multiple-apirules-targeting-same-workload.md) documentation before proceeding.
```

Comment:
```suggestion
> If multiple APIRules target the same workload, you must perform an additional migration step to avoid traffic disruption. This step involves creating an additional, temporary AuthorizationPolicy before applying the first migrated APIRule `v2`. For detailed instructions, refer to the [Migrate multiple APIRules `v1beta1` targeting same workload](./01-90-migrate-multiple-apirules-targeting-same-workload.md) documentation before proceeding.
```